lib/public/views/Tags/Create/index.js

```diff
@@ -33,17 +33,17 @@ const createTagScreen = (model) => {
         }, [
             h('h2', 'Create Tag'),
 
-            h('label.form-check-label.f4.mt2', { for: 'text' }, 'Text'),
-            h('input#text.form-control.w-25.mb2', {
-                placeholder: 'Enter the tag content...',
-                minlength: 2,
-                maxlength: 20,
-                value: text,
-                oninput: (e) => {
-                    model.tags.text = e.target.value;
-                },
-            }, text),
             model.isAdmin() && [
```

Comment:
As you mentioned, you should add an error message stating that the user do not have rights to create a tag, because here for a non admin it looks a bit like a bug